### PR TITLE
feature/gh-18 jmeter-plugins

### PR DESCRIPTION
**Pull request by mrmeyers99**
I added a command to install the plugin manager, a variable with plugins to install, and also copied jar from a dependency folder into the jmetere/lib folder.  After this I was able to get everything to work. I tried to avoid copying files over but I couldn't get `search_paths` to work.

### CODE REVIEW

File: cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+rm -rf ${REPORT_FOLDER}
```
Comment (mrmeyers99) [inline cleanup.sh:2]: Adding this cleanup job deletes files owned by root which causes problems on runners that don't run as root.

**Comment by mrmeyers99** [comment]
@rbhadti94 Could you take a look at this when you get a moment?

**Comment by rbhadti94** [comment]
Hey @mrmeyers99, thanks for the contribution! Yes will take a look at this :)

**Comment by rbhadti94** [comment]
Thanks for the contribution @mrmeyers99. This has now been v0.4.0. Please let me know if this works fine. Can always make changes on top of this.

https://github.com/rbhadti94/apache-jmeter-action/releases/tag/v0.4.0

**Comment by mrmeyers99** [comment]
I was able to run my action using the `v0.4.0` tag in the repo.  I'll let yo know if I run into any other issues.  Thanks!

**Pull request merged**